lib/buildChunkGraph.js

```diff
@@ -327,15 +327,16 @@ const visitModules = (
 			c = cgi.chunkGroup;
 		}
 
-		// 2. We store the Block+Chunk mapping as dependency for the chunk
-		let deps = chunkDependencies.get(chunkGroup);
-		if (!deps) chunkDependencies.set(chunkGroup, (deps = []));
+		// 2. We store the Block + Chunk Group mapping as dependency
+		// for the chunk group which is set in processQueue
+		let deps = chunkGroupDependencies.get(chunkGroup);
+		if (!deps) chunkGroupDependencies.set(chunkGroup, (deps = []));
 		deps.push({
 			block: b,
 			chunkGroup: c
 		});
 
-		// 3. We create/update the chunk group info
+		// 3. We enqueue the cunk group info creation/updating
```

Comment:
```suggestion
		// 3. We enqueue the chunk group info creation/updating
```

It's enqueued for performance reasons. We group multiple items together to save duplicate work. In addition to that some work need to be done after all modules for the current chunk groups has been walked to get module indices correct.